web/src/modules/topic/components/CriteriaTable/CriteriaTable.styles.tsx

```diff
@@ -2,8 +2,8 @@ import { css } from "@emotion/react";
 import styled from "@emotion/styled";
 import { Button, IconButton } from "@mui/material";
 
-import { AddNodeButton } from "../AddNodeButton/AddNodeButton";
-import { nodeWidth } from "../EditableNode/EditableNode.styles";
+import { AddNodeButton } from "../Node/AddNodeButton";
```

Comment:
FYI @Nezzely this last commit tries to reorganize the `modules/topic/components/` directory since it was getting pretty big. I don't think it's perfect but it seems maybe better for now? Lemme know if you have any suggestions